### DISCUSSION TITLE
Getting actual member List

### DISCUSSION THREAD

**Discussion by Ralf-Os** [tomer8007/kik-bot-api-unofficial#213] [Q&A]
Hi,
how can I grab an actual member list of a group? It seems, only groupSysMessage sends member lists. But that is only fired on joining a group with the bot. 
My problem is to keep track of members in my database, I often get old entrys that are no longer in the group.

**Comment by StethoSaysHello** [comment]
Hello! Just letting you know that _publicly_ answering this question in detail would encourage thot bots that could post mass advertisements (as is discussed [here](https://github.com/tomer8007/kik-bot-api-unofficial/issues/207#issuecomment-939576858)), so you may not get a clear answer to this question here. Best of luck to you!
